data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1918 @@
+			`	"Much assistance, we require, Captain," they say. "A transport for <destination>, we need. For <bunks> of us, and <cargo>. Of great urgency, this task is."`
+			`	Without you even noticing it, an Arach agent has also gotten close, and after the Kimek is done speaking, says, "Attacking several factory districts and government buildings, the terrorists are, so at once we must go. <payment>, you will receive, if by <date> we arrive."`
+			choice
+				`	"I'd be glad to help. Bring your colleagues and the cargo to my ship."`
+					goto accept
+				`	"Terrorists? Can't the Heliarchs on that planet take care of it themselves?"`
+				`	"You'll have to find someone else. I'm not heading there right now."`
+					decline
+			`	"As I said, hit many government buildings, they have," the Arach explains. "Damaged several garrisons, the attacks did, and crippled the local agents' immediate response capabilities, they have. Not to mention, a very populous and busy world, <planet> is, so a more difficult task, locating and arresting the criminals is."`
+			`	"To keep the peace, our job is," the Kimek adds. "If to assist in achieving that goal, you wish, transport us there, you should."`
+			choice
+				`	"Okay. Bring your colleagues and the cargo to my ship."`
+				`	"You'll have to find someone else. I'm not heading there right now."`
+					decline
+			label accept
+			`	They thank you for your cooperation, and call in a device for the rest of the agents to come to your ship, along with their cargo. After securing the last of the containers in your hold, you show them to their bunks, and head to the cockpit.`
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		payment 420000
+		dialog `You're told to open your hatch as soon as you land, and when you do, the Heliarch agents storm out of your ship while a separate group that was waiting for you unloads the cargo. A few seconds later you see that <payment> have been added to your account.`
+
+mission "Heliarch Containment 2"
+	name "Pick Up Injured Soldiers"
+	description "Land on <stopovers>, to pick up the <bunks> Heliarch soldiers in need of medical attention, and transport them to <destination> by <date>."
+	deadline
+	passengers 21
+	to offer
+		has "Heliarch Containment 1: done"
+		random > 35
+	source
+		near "Bloptab" 1 3
+	stopover "Delve of Bloptab"
+	destination "Ahr"
+	on offer
+		conversation
+			`You're stopped by a group of Heliarch agents as you enter the spaceport, most of them Arachi.`
+			`	"Injured, many colleagues of ours have been. On <stopovers>, they are," one of them says. "Dealing with containing criminal operations, they were, surprised them with an attack amidst civilians, the rogues did. Given the circumstances, dangerous for them, it might be, if treated within hospitals there, they are."`
```

Comment:
```suggestion
			`	"Injured, many colleagues of ours have been. On <stopovers>, they are," one of them says. "Dealing with containing criminal operations, they were, and surprised them with an attack amidst civilians, the rogues did. Given the circumstances, dangerous for them, it might be, if treated within hospitals there, they are."`
```